pmm-app/src/pmm-settings/components/SignUp/FieldAdapters/FieldAdapters.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { cx } from 'emotion';
+import {
+  Input,
+  useTheme,
+} from '@grafana/ui';
+import { getStyles } from './FieldAdapters.styles';
+// TODO (nicolalamacchia): use Grafana's components once migration to Grafana v7 is complete
+import { Checkbox } from './Checkbox';
+import { Field } from './Field';
+
+export const InputFieldAdapter = ({
+  input, className, label, meta, ...props
+}) => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
+
+  return (
+    <Field label={label}>
+      <>
```

Comment:
I think if you set the children prop in the FieldProps to ReactNode instead of ReactElement you don't need the Fragment. Anyway I understand that the Field is just temporary. And I wonder once we migrate to Grafana 7 Field component if it's better to put these Adapters on the shared or refactor the existing ones in there.

Reply:
That's because the `Field` components comes directly from Grafana. I moved it temporarily here in order to avoid conflicts between different Grafana versions. It will disappear once we migrate to Grafana 7, for this reason I prefer to leave it as it is for now :)